packages/redux/src/categories/types/state.types.ts

```diff
@@ -1,6 +1,8 @@
 import type { BlackoutError, Category } from '@farfetch/blackout-client';
 import type { CombinedState } from 'redux';
 
+export type CategoryId = string;
```

Comment:
Question: Why is this type necessary? There is the `Category['id']` type which is a number and not a string.